scanners/port_scanner.py

```diff
@@ -0,0 +1,25 @@
+import nmap
+import os
+import time
+
+
+class PortScanner():
+
+    def __init__(self):
+        # This is needed for nmap static library to be added to the path
+        os.environ['PATH'] = os.environ['PATH'] \
+            + ':' + os.environ['LAMBDA_TASK_ROOT'] \
+            + '/bin/nmap-standalone/'
+    
+    def scanTCP(self, host):
+        # TODO: There is an async scan option in python-nmap, use that instead
+        nm = nmap.PortScanner()
+        isSudo = False
+        nmap_arguments = ("-v -Pn -sT -sV --script=banner " +
+                          "--top-ports 1000 --open -T4 --system-dns"
+                          )
+        results = nm.scan(host, arguments=nmap_arguments, sudo=isSudo)
+        time.sleep(120)
```

Comment:
What is the purpose of the 120 sec sleep here?  I would think we should add  comments at least, but I have to assume that this functionally is a blocking operation if you're mentioining async scans  as a future improvement.

Reply:
This was only used as a PoC, indeed it's a bad implementation. The `python-nmap` library supports async invocation of `nmap`, I will re-write it using that.